doc/nrf/ug_bootloader.rst

```diff
@@ -7,212 +7,185 @@ Secure bootloader chain
+* If you want the bootloader to support firmware updates for both itself and the application, then use the :ref:`two-stage, upgradable bootloader <upgradable_bootloader>` solution.
+
+Both single-stage and two-stage architectures are built upon the same chain of trust concept.
 
 Chain of trust
 **************
 
-A secure system depends on building and maintaining a chain of trust through all layers in the system.
-Each step in this chain will guarantee that the next step can be trusted to have certain properties, because any illegal modification of a following step will be detected and the process halted.
-The trustworthiness of each layer is guaranteed by the previous layer, all the way back to a property in the system referred to as a root of trust (RoT).
+A secure system depends on building and maintaining a chain of trust through all the layers in the system.
+Each step in this chain guarantees that the next step can be trusted to have certain properties because any unauthorized modification of a subsequent step will be detected and the process halted.
+The trustworthiness of each layer is guaranteed by the previous layer, all the way back to a property in the system referred to as *root of trust* (RoT).
+
+You can compare a chain of trust to the concept of a door:
 
-You can compare a chain of trust to the concept of a door.
-You trust a door because you trust the lock because you trust the key because the key is in your pocket.
-If you lose this key the chain of trust unravels and you no longer trust this door.
+1. You trust a door because you trust the lock.
+#. You trust the lock because you trust the key.
+#. You trust the key because it is in your pocket.
+
+If you lose this key, the chain of trust unravels and you can no longer trust this door.
 
 It is important to note that a key alone is not a RoT, although a RoT may include a key.
-A RoT will consist of both hardware, software, and data components that must always behave in an expected manner, because any misbehavior cannot be detected.
+A RoT will consist of hardware, software, and data components that must always behave as expected because any misbehavior cannot be detected.
 
 .. _ug_bootloader_architecture:
 
 Architecture
 ************
 
-The secure bootloader chain consists of a sequence of images that are booted one after the other.
-To establish a root of trust, the first image verifies the signature of the next image (which can be the application or another bootloader image).
+In the context of the bootloader, a step in the chain of trust is the boot of a specific image.
+As such, the secure bootloader chain consists of a sequence of images that are booted one after the other.
+
+To establish a root of trust, the first image verifies the signature of the next image, which can either be an application or another bootloader image.
```

Comment:
What is `a root of trust` and how is it different than `a chain of trust`?

Reply:
This is covered a bit a little earlier in the Chain of Trust section:

> The trustworthiness of each layer is guaranteed by the previous layer, all the way back to a property in the system referred to as *root of trust* (RoT)...It is important to note that a key alone is not a RoT, although a RoT may include a key.
A RoT will consist of hardware, software, and data components that must always behave as expected because any misbehavior cannot be detected.

But there is no succinct definition anywhere. Since this is an external term used with things like [PSA](https://www.psacertified.org/blog/what-is-a-root-of-trust/), I'm not sure its within our scope to define it, but we could perhaps link to PSA's definition for reference?